celery_pubsub/pubsub.py

```diff
@@ -84,6 +92,20 @@ def _topic_to_re(topic: str) -> re.Pattern[str]:
 _pubsub_manager: PubSubManager = PubSubManager()
 
 
+def subscribe_to(topic: str) -> typing.Callable:
```

Comment:
```suggestion
def subscribe_to(topic: str) -> typing.Callable[[typing.Callable[P, R]], Task[P, R]]:
```

Reply:
fix `typing.Callable[P, R]` to `typing.Callable[[P], R]`